content/community-projects.md

```diff
@@ -53,7 +53,7 @@ _We are currently working on extracting a Go SDK out of the [Duffle codebase][du
 ---
 
 Have you written an SDK or tool that works with the CNAB spec? Submit a PR
-to <https://github.com/deislabs/cnab.io> and we will list it here!
+to <https://github.com/cnabio/cnab.io> and we will list it here!
 
 [cnab-community]: https://github.com/deislabs/cnab-community
```

Comment:
Do we want to update all references? Like this link here should be https://github.com/cnabio/community maybe we should expand the search for `deislabs/cnab` and we will find a few more old URLs from before we moved the repos.

Same for the duffle link below too.

Reply:
Good call; thank you!  All should be updated I think.  Will file a few follow-up tickets (remove/replace link to deprecated example bundles repo, etc.)